README.md

```diff
@@ -3,8 +3,31 @@
 This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
 
 ## Setup
-- `git clone https://github.com/thewca/worldcubeassociation.org` - Clone this repo! (And navigate into it, `cd worldcubeassociation.org`)
-- `(cd WcaOnRails; bundle install && bundle exec pre-commit install) && git config pre-commit.ruby "scripts/ruby_in_wca_on_rails.sh"` - Set up git pre-commit hook. Optional, but very useful.
+- Clone this repo! (And navigate into it)
+  ```
+  git clone https://github.com/thewca/worldcubeassociation.org
+  cd worldcubeassociation.org
+  ```
+- Ensure you have the correct [Ruby version](./.ruby_version) installed. We recommend using a Ruby version manager like [rvm](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv). To install using rvm:
+  ```
+  rvm install 2.5.3
+  rvm use 2.5.3
```

Comment:
```suggestion
  rvm use `cat .ruby-version`
```

Reply:
Using rbenv it automatically detects the ruby version it should use from the `.ruby-version` file; is it also the case with rvm?

If yes then it may be better to just indicate the user to check that the selected version of ruby is correct.

Reply:
According to the [documentation](https://rvm.io/workflow/projects#supported-files), it does. I'll update this section.